nemo_skills/mcp/adapters.py

```
@@ -81,38 +92,32 @@ def format(self, tool_call: ChatCompletionMessageToolCall, result):
+
+
+def get_tool_details_by_completion_type(tool_call, completion_type: CompletionType):
+    if completion_type == CompletionType.chat:
+        tool_name = tool_call["function"]["name"]
+        tool_args = tool_call["function"]["arguments"]
+    elif completion_type == CompletionType.responses:
+        assert tool_call["type"] == "function_call", "Tool call must be a function call"
+        tool_name = tool_call["name"]
+        tool_args = tool_call["arguments"]
+    else:
+        raise ValueError(f"Unsupported completion type for tool call: {completion_type}")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)